### PR TITLE
#150 a11y pass on Audio player

### LINKED ISSUE #150: Buttons do not have an accessible name
Specifically the audio play button. https://web.dev/button-name/?utm_source=lighthouse&utm_medium=devtools

### PR DESCRIPTION
**Pull request by brandonhundt**
Closes #150

- Adds aria-labels on all the buttons and links in the player

## To Review

- [ ] Use the Preview link located in the Now comment below.

> ...or...

- [ ] Checkout Branch.
- [ ] Run `yarn`.
- [ ] Run `yarn dev:start`.
- [ ] Go to [localhost:3000](https://localhost:3000).

> ...then...
- [ ] Go to any story with audio, run Lightbox, and ensure all links are described appropriately

### CODE REVIEW

File: components/AudioPlayer/AudioPlayer.tsx [hunk 1]
```diff
@@ -244,6 +248,7 @@ export const AudioPlayer = ({
               component="a"
               href={url}
               download={audioDownloadFilename}
+              aria-label="Download"
```
Comment (rpeterman-gp) [inline components/AudioPlayer/AudioPlayer.tsx:251]: This should be "Download audio" to be descriptive of what is being downloaded.

File: components/AudioPlayer/AudioPlayer.tsx [hunk 2]
```diff
@@ -218,6 +220,7 @@ export const AudioPlayer = ({
             {!!embeddedPlayerUrl && (
               <IconButton
                 className={classes.embedBtn}
+                aria-label="Embed Code"
```
Comment (rpeterman-gp) [inline components/AudioPlayer/AudioPlayer.tsx:223]: This should probably update based on player state: `aria-label={embedCodeShown ? 'Hide embed code' : 'Show embed code'}`

File: components/AudioPlayer/AudioPlayer.tsx [hunk 3]
```diff
@@ -201,6 +202,7 @@ export const AudioPlayer = ({
               </Box>
               <Box className={classes.volumeControls}>
                 <IconButton
+                  aria-label="Toggle Mute"
```
Comment (rpeterman-gp) [inline components/AudioPlayer/AudioPlayer.tsx:205]: This should probably update based on player state: `aria-label={muted ? 'Mute' : 'Unmute'}`

File: components/AudioPlayer/AudioPlayer.tsx [hunk 4]
```diff
@@ -172,6 +172,7 @@ export const AudioPlayer = ({
         <div ref={rootElm} className={rootClasses}>
           <IconButton
             className={playBtnClasses}
+            aria-label="Play"
```
Comment (rpeterman-gp) [inline components/AudioPlayer/AudioPlayer.tsx:175]: This should probably update based on player state: `aria-label={playing ? 'Pause' : 'Play'}`

**Review by rpeterman-gp — CHANGES_REQUESTED** [review]

File: components/AudioPlayer/AudioPlayer.tsx (hunk 1 above)
Comment (brandonhundt) [reply components/AudioPlayer/AudioPlayer.tsx]: Fixed!

File: components/AudioPlayer/AudioPlayer.tsx (hunk 3 above)
Comment (brandonhundt) [reply components/AudioPlayer/AudioPlayer.tsx]: Fixed!

File: components/AudioPlayer/AudioPlayer.tsx (hunk 4 above)
Comment (brandonhundt) [reply components/AudioPlayer/AudioPlayer.tsx]: Fixed!

File: components/AudioPlayer/AudioPlayer.tsx (hunk 2 above)
Comment (brandonhundt) [reply components/AudioPlayer/AudioPlayer.tsx]: Fixed!

**Review by rpeterman-gp — APPROVED** [review]

**Pull request merged**